client/components/payments-activity/style.scss

```
@@ -10,45 +26,121 @@
 	}
 }
 
-.payments-activity-data {
-	display: flex;
-	padding: 24px;
+.wcpay-payments-activity-data {
+	display: grid;
+	grid-template-columns: 1fr 1fr;
+	width: 100%;
 
 	@include breakpoint( '<660px' ) {
-		flex-direction: column;
+		grid-template-columns: 1fr;
+		padding: 24px;
 	}
 
-	.total-payments-volume {
-		display: flex;
-		flex: 2;
-		flex-direction: column;
-		align-items: flex-start;
+	.wcpay-payments-data-highlights {
+		display: grid;
+		grid-template-columns: 1fr 1fr;
 
-		&__heading {
-			font-size: 16px;
-			font-weight: 600;
-			line-height: 16px;
-			color: $gray-900;
+		@include breakpoint( '<660px' ) {
+			grid-template-columns: 1fr;
 		}
 
-		&__body {
-			font-size: 24px;
-			font-weight: 600;
-			color: $gray-900;
-			line-height: 32px;
-			margin-top: 14px;
+		&__item {
+			padding: 24px;
+			border-left: 1px solid $gray-200;
+			display: flex;
+			flex-direction: column;
+			justify-content: flex-start;
+			align-items: flex-start;
+			min-height: 129px;
+
+			&:hover {
+				.wcpay-payments-data-highlights__item__wrapper__report-link {
+					visibility: visible;
+				}
+			}
+
+			&__label {
+				@include label-styles;
+				margin: 0 0 8px 0;
+			}
+
+			&__wrapper {
+				display: flex;
+				flex-direction: column;
+				align-items: flex-start;
+				width: 100%;
+
+				@include breakpoint( '<660px' ) {
+					flex-direction: row;
+					justify-content: space-between;
+					align-items: center;
+				}
+
+				&__amount {
+					@include amount-styles;
+					margin: 0 0 8px 0;
+
+					@include breakpoint( '<660px' ) {
+						margin: 0;
+					}
+				}
+
+				&__report-link {
+					text-decoration: none;
+					font-size: 12px;
+					visibility: hidden;
+
+					@include breakpoint( '<660px' ) {
+						min-height: unset;
+						visibility: visible;
+					}
+				}
```

💭 We can support keyboard navigation to the `View report` links if we use `opacity` rather than `visibility`.

(Note this suggestion uses the `a` locator as it target the `<Link />` as per previous comment)

```suggestion
				a {
					text-decoration: none;
					font-size: 12px;
					opacity: 0;

					&:focus {
						opacity: 1;
					}

					@include breakpoint( '<660px' ) {
						min-height: unset;
						opacity: 1;
					}
				}
```

---

Should those hover styles be in a shared link component? i.e. not specific to this UI

---

> Should those hover focus styles be in a shared link component?

This visibility/opacity styling is to show/hide the link on hover/focus. The underlying focus styling (outline) of the link component remains.

---

> This visibility/opacity styling is to show/hide the link on hover/focus.

Is that a common need or special to this UI?

---

Special to this UI, I believe. If not, for sure we could have a more generic component.

---

This is a great improvement point. I have fixed this here d53c1de .